read_roi/_read_roi.py

```diff
@@ -316,9 +316,13 @@ def extract_basic_roi_data(data):
         overlayFontSize = get_uint16(data, hdr2Offset + HEADER_OFFSET['OVERLAY_FONT_SIZE'])
         imageOpacity = get_byte(data, hdr2Offset + HEADER_OFFSET['IMAGE_OPACITY'])
         imageSize = get_uint32(data, hdr2Offset + HEADER_OFFSET['IMAGE_SIZE'])
+        # abb, read group from file
```

Comment:
I don't understand the need for this comment; this is one of many bytes being loaded all in the same way, and there is no need to comment on this particular one. Not sure why it begins with "abb," either. What does this mean?